task/control-plane-agent/src/mgs_compute_sled.rs

```diff
@@ -956,6 +956,29 @@ impl SpHandler for MgsHandler {
         }
     }
 
+    fn component_cancel_pending_active_slot(
+        &mut self,
+        component: SpComponent,
+        slot: u16,
+        persist: bool,
+    ) -> Result<(), SpError> {
+        ringbuf_entry_root!(Log::MgsMessage(
+            MgsMessage::ComponentCancelPendingActiveSlot {
+                component,
+                slot,
+                persist,
+            }
+        ));
+        match component {
+            SpComponent::HOST_CPU_BOOT_FLASH => {
+                Err(SpError::RequestUnsupportedForComponent)
+            }
```

Comment:
It feels a little strange to have the check for `HOST_CPU_BOOT_FLASH` in all three specific mgs files since we also have the fallback in `mgs_common.rs` . I think this should probably just be removed.